examples/FCCee/fullSim/caloNtupleizer/analysis.py

```diff
@@ -72,9 +72,8 @@ def run(self):
                 dict_outputBranchName_function["%s_eta"%cellBranchName] = "CaloNtupleizer::getCaloHit_eta(%s)"%cellBranchName
                 dict_outputBranchName_function["%s_energy"%cellBranchName] = "CaloNtupleizer::getCaloHit_energy(%s)"%cellBranchName
                 if args.useGeometry:
-                    dict_outputBranchName_function["%s_phiBin"%cellBranchName] = "CaloNtupleizer::getCaloHit_phiBin(%s)"%cellBranchName
+                    dict_outputBranchName_function["%s_thetaBin"%cellBranchName] = "CaloNtupleizer::getCaloHit_thetaBin(%s)"%cellBranchName
```

Comment:
`phiBin` is still relevant, but its new name is `module` (see [this](https://github.com/key4hep/k4geo/blob/main/FCCee/ALLEGRO/compact/ALLEGRO_o1_v02/ECalBarrel_thetamodulemerged.xml#L91) and `etaBin` was replaced by `thetaBin`. We should 1) add a `get_moduleIdx` method in CaloNtupleizer (and rename accordingly the *Bin to *Idx everywhere to keep things consistent), 2) replace *_phiBin by *_moduleIdx in analysis.py and replace etaBin by thetaBin in analysis.py

Reply:
Done